src/hooks/useTranslate.ts

```diff
@@ -0,0 +1,17 @@
+import { Translations } from '../types';
+
+const useTranslate = (translations?: Translations) => {
```

Comment:
Can we move this to utils and change the name format? `useXXX` implies this is a custom hook but it's not